packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx

```diff
@@ -71,7 +71,7 @@ export const NftWidget: React.FC<NftWidgetProps> = ({
     const buttonSize = size === 'small' ? 'medium' : 'large'
     const buttonColumnSpan = size === 'small' ? 1 : 2
     const BuyNow = ({ buyNowPrice }: { buyNowPrice?: number }) =>
-      buyNowPrice ? (
+      buyNowPrice && ~~buyNowPrice ? ( // TODO: remove `&& ~~buyNowPrice` when the payload type of buyNowPrice is fixed and it doesn't return strings anymore
```

Comment:
That's a nice hack, but can we just do a `Number(buyNowPrice)`? It wasn't clear at all to me - I had to google what actually `~~` does 😅 . I don't think it's common javascript knowledge.